connexion/decorators/validation.py

```diff
@@ -63,6 +82,20 @@ def make_type(value, type_literal):
                 converted = v
             converted_params.append(converted)
         return converted_params
+    elif param_type == 'object':
+        converted_params = {}
+        if 'properties' in param_schema and param_schema['properties']:
+            for k, v in value.items():
+                try:
+                    converted_params[k] = make_type(v, param_schema['properties'][k]['type'])
```

Comment:
will this work for nested properties?

Reply:
seems yes. the tests pass, and they use nested properties. or you see an additional test case missing?

Reply:
forget it, the tests don't pass. will fix it.